staging/src/k8s.io/apiserver/pkg/cel/limits.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cel
+
+const (
+	// MaxRequestSizeBytes is the largest request that will be accepted is 3MB
+	// TODO(DangerOnTheRanger): wire in MaxRequestBodyBytes from apiserver/pkg/server/options/server_run_options.go to make this configurable
+	MaxRequestSizeBytes = int64(3 * 1024 * 1024)
```

Comment:
we shouldn't expose a public const with a TODO to make it configurable :)

when this was private, the TODO was more reasonable, but we don't want to pick up dependents on this

notably, MaxRequestBodyBytes is not configurable via flags or file-based config in kube-apiserver or apiserver... only someone modifying the server config programatically could change that

Reply:
Here we still have the default value of `MaxRequestSizeBytes`. I am thinking of exporting the default value here and let the implementation (still under apiextensions-apiserver) wire in the default.

Reply:
I left it exported as `DefaultMaxRequestSizeBytes` because later CEL for built-in types would still consume this set of limits.